agent/inject.js

```diff
@@ -21,10 +21,10 @@ module.exports = function(hook: Hook, agent: Agent) {
       agent.setReactInternals(id, helpers);
       helpers.walkTree(agent.onMounted.bind(agent, id), agent.addRoot.bind(agent, id));
     }),
-    hook.sub('root', ({renderer, element}) => agent.addRoot(renderer, element)),
-    hook.sub('mount', ({renderer, element, data}) => agent.onMounted(renderer, element, data)),
-    hook.sub('update', ({renderer, element, data}) => agent.onUpdated(element, data)),
-    hook.sub('unmount', ({renderer, element}) => agent.onUnmounted(element)),
+    hook.sub('root', ({renderer, internalInstance}) => agent.addRoot(renderer, internalInstance)),
```

Comment:
I think other than this, which is not a update per se, all other review comments have been fixed.